### PR TITLE
[fix](npe) fix load be id npe

### PR DESCRIPTION
**Pull request by yujun777**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yujun777** [comment]
run buildall

File: cloud/src/meta-service/codec.cpp [hunk 1]
```diff
@@ -17,6 +17,8 @@
 
 #include "codec.h"
 
+#include <cstdint>
```
Comment (gavinchou) [inline cloud/src/meta-service/codec.cpp:20]: why do we need this?

**Review by gavinchou — CHANGES_REQUESTED** [review]

**Comment by yujun777** [comment]
run p0

**Comment by yujun777** [comment]
run p0

File: cloud/src/meta-service/codec.cpp (hunk 1 above)
Comment (yujun777) [reply cloud/src/meta-service/codec.cpp:20]: gcc need this. otherwise unknown type:  uint64_t

**Comment by yujun777** [comment]
run p0

**Comment by yujun777** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by liaoxin01 — APPROVED** [review]

**Pull request merged**